src/libs/WorkflowUtils.ts

```diff
@@ -82,14 +82,36 @@ type ConvertPolicyEmployeesToApprovalWorkflowsParams = {
      * Email of the default approver for the policy
      */
     defaultApprover: string;
+
+    /**
+     * Email of the first approver in current edited workflow
+     */
+    firstApprover?: string;
+};
+
+type PolicyConversionResult = {
+    /**
+     * List of approval workflows
+     */
+    approvalWorkflows: ApprovalWorkflow[];
+
+    /**
+     * List of available members that can be selected in the workflow
+     */
+    availableMembers: Member[];
+
+    /**
+     * Emails that are used as approvers in currently configured workflows
+     */
+    usedApproverEmails: string[];
 };
 
 /** Convert a list of policy employees to a list of approval workflows */
-function convertPolicyEmployeesToApprovalWorkflows({employees, defaultApprover, personalDetails}: ConvertPolicyEmployeesToApprovalWorkflowsParams): ApprovalWorkflow[] {
+function convertPolicyEmployeesToApprovalWorkflows({employees, defaultApprover, personalDetails, firstApprover}: PolicyConversionParams): PolicyConversionResult {
     const approvalWorkflows: Record<string, ApprovalWorkflow> = {};
 
-    // Keep track of how many times each approver is used to detect approvers in multiple workflows
-    const approverCountsByEmail: Record<string, number> = {};
+    // Keep track of used approver emails to display hints in the UI
+    const usedApproverEmails = new Set<string>();
```

Comment:
Is there any reason why we don't use an empty string?

```suggestion
    const usedApproverEmails = [];
```

Reply:
We could do it but there will be duplicates in the array this way, that's why I believe a Set is a better fit here

Reply:
Ahh good point. Thanks